build/scripts/build.ps1

```diff
@@ -129,10 +129,6 @@ function Run-MSBuild([string]$buildArgs = "", [string]$logFile = "", [switch]$pa
         $args += " /bl:$logFile"
     }
 
-    if ($cibuild) { 
-        $args += " /p:PathMap=`"$($repoDir)=q:\roslyn`" /p:Feature=pdb-path-determinism" 
-    }
```

Comment:
Why did we remove this?

Reply:
It's not needed -- the targets know when we are building in CI and set the PathMap accordingly now. "pdb-path-determinism" isn't used by the compiler anymore, right?

---
In reply to: [147460707](https://github.com/dotnet/roslyn/pull/22850#discussion_r147460707) [](ancestors = 147460707)

Reply:
Gotcha. See it now in the core target files

---
In reply to: [147761078](https://github.com/dotnet/roslyn/pull/22850#discussion_r147761078) [](ancestors = 147761078,147460707)